### DISCUSSION TITLE
Skywalking Support For Vertica Database

### DISCUSSION THREAD

**Discussion by AjinThankachan** [apache/skywalking#8817] [Q&A]
Hi ,
I wish to try and add vertica as the backend for skywalking.Is there any plugin or support available as of now? If i need to add and create a support how we can configure it. Can some one please provide an input on this so that i can try and add that support.
Thanks in advance for your support and guideces.

**Comment by wu-sheng** [comment]
No, I think we only have ElasticSearch, OpemSearch, InfluxDB, Apache IOTDB, H2/MySQL/PostgreSQL for now.

**Reply by AjinThankachan** [reply]
Thanks for your prompt response. If we wanted to add as a new feature is it possible to add and test it. Is there any specific requirement or dependency there

**Reply by wu-sheng** [reply]
Storage options are pluggable, but you need to implement methods required by core.

**Reply by AjinThankachan** [reply]
Thanks a lot.Will try to check further.

**Reply by wu-sheng** [reply]
This was the IoTDB plugin added, https://github.com/apache/skywalking/pull/7766. You could take a reference, of course, the APIs have some changes due to v9 being the mainstream now.
